### DISCUSSION TITLE
Budibase Template giving error Document update conflict

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#11525] [Help] [answered]
Hi

i installed budibase template for IT ticketing system and published it without making any chnages.

it is giving error Document update conflict as shown in the attached screenshot. Any fix?

![20230815_094245](https://github.com/Budibase/budibase/assets/130876667/9806b87d-262b-4bb1-bd14-a8441a08111b)

**Comment by andz-bb** [comment]
Hey @sghriyadh, could you check that the Views in that app are loading properly when you're in the Data section?

<img width="1029" alt="image" src="https://github.com/Budibase/budibase/assets/110921612/028ca8bc-23a6-4fb1-8690-7f37afd436ce">

If those are OK, could you try republishing the apps and check if the errors are still present? Thanks

**Comment by lesleykie** [comment]
Hi I get this identical issue : Views are not accessible, and when published the app also gives: Document update conflict ... 
I reinstalled budibase, but I am sure there is a much smarter way to fix this! :) :) (since that did not solve it) :)

**However**: re-publishing got rid of the "Document update conflict ", just the **views** just won't load on the screen.
Clicking on any view just stalls the backend.

I hope you would be able to point my in a direction?
THX!

**Comment by melohagan** [comment, marked as answer]
Created an issue here: https://github.com/Budibase/budibase/issues/12496
